flaskdeid/annotation.py

```diff
@@ -0,0 +1,110 @@
+"""Module for standardizing and combining annotations"""
+class Annotation(object):
+    def __init__(self, origin):
+        self.origin = origin
+        self.start = None
+        self.end = None
+        self.score = None
+        self.type = None
+        self.text = None
+
+    def empty(self):
+        return not (self.start and self.end and self.text)
+
+    def from_medlp(medlp):
+        ann = Annotation('medlp')
+        ann.start = medlp.get('BeginOffset')
+        ann.end = medlp.get('EndOffset')
+        ann.score = medlp.get('Score')
+        ann.type = medlp.get('Type')
+        ann.text = medlp.get('Text')
+        return ann
+
+    def from_hutchner(hutchner):
```

Comment:
Same as above.

Reply:
Per our discussion, this was fixed with your Factory change.